NextcloudTalk/NCRoomsManager.m

```diff
@@ -528,6 +530,9 @@ - (void)startChatInRoom:(NCRoom *)room
     
     if (!_chatViewController || ![_chatViewController.room.token isEqualToString:room.token]) {
         _chatViewController = [[NCChatViewController alloc] initForRoom:room];
+        if (_highlightMessageDict && [[_highlightMessageDict objectForKey:@"token"] isEqualToString:room.token]) {
+            _chatViewController.highlightMessageId = [[_highlightMessageDict objectForKey:@"messageId"] integerValue];
```

Comment:
This needs to be resetted as well, otherwise the message will always be highlighted whenever a room is entered:

https://user-images.githubusercontent.com/1580193/179426389-734d39cd-db0f-4973-813a-56d54794f9e6.MP4

Reply:
Yep 😅